### DISCUSSION TITLE
Osram Smart+ Mini AC0251100NJ/AC0251600NJ/AC0251700NJ sending wrong Actions being sent

### DISCUSSION THREAD

**Discussion by Sanctustalio** [Koenkk/zigbee2mqtt#13432] [Devices]
I've just set a pi zero running Zigbee2MQTT, I've got 4 control devices. 
One Sonoff SNZB-01 and 3 of the above Osram Lightify Smart+ Minis. The Sonoff device is working perfectly with home assistant automations but the minis all don't seem to activate any actions correctly with a clean installation.

On checking the log output from Zigbee2MQTT it seems as if the actions being sent don't align with what they should be.
[https://www.zigbee2mqtt.io/devices/AC0251100NJ_AC0251600NJ_AC0251700NJ.html#action-enum](url)
"The possible values are: up, up_hold, up_release, down_release, circle_release, circle_hold, down, down_hold, circle_click."

In the logs I'm getting
Up button pressed
"MQTT publish: topic 'zigbee2mqtt/Lightify Switch Mini 2/action', payload 'on'"

Center button
MQTT publish: topic 'zigbee2mqtt/Lightify Switch Mini 2/action', payload 'brightness_move_to_level'
followed by
MQTT publish: topic 'zigbee2mqtt/Lightify Switch Mini 2/action', payload 'color_temperature_move'

Down button
MQTT publish: topic 'zigbee2mqtt/Lightify Switch Mini 2/action', payload 'off'

After some playing around in the settings one of the minis is playing ball but I'm unsure what changed it to behave correctly but originally all 3 defaulted to the above scheme.

With further investigation it seems that the legacy option in settings (specific) is defaulting to False instead of True and if forced to True the actions are correct.

**Comment by Koenkk** [comment]
Updated the actions ,it should list the non-legacy ones instead of the legacy ones. Docs will be updated with the next release.
